ansible_collections/arista/avd/docs/installation/requirements.md

```diff
@@ -47,3 +40,14 @@ pip3 install -r ansible-avd/ansible_collections/arista/avd/requirements.txt
 
 !!! warning
     Depending of your operating system settings, `pip3` might be replaced by `pip`.
+
+### Ansible Configuration INI file
```

Comment:
The file is called `ansible.cfg` so calling it `INI` is a bit confusing.

Reply:
Agreed, thank you